CHANGELOG.md

```diff
@@ -42,7 +42,16 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ## [Unreleased]
 
 ## [v0.1.0](https://github.com/umee-network/peggo/releases/tag/v0.1.0) - 2021-12-18
-
 ### Features
 
 - Initial release!!!
+
+## [#104](https://github.com/umee-network/peggo/pull/104) Chunk claims messages
+
+### Bug Fixes
+
+- Claims are split into chunks of 10 to avoid hitting request limits.
+
+### Improvements
+
+- Changed timeout for broadcasting TXs to Umee to 60s to match that of the official Gravity Bridge.
```

Comment:
```suggestion
### Bug Fixes

- [#104] Claims are split into chunks of 10 to avoid hitting request limits.

### Improvements

- [#104] Changed timeout for broadcasting TXs to Umee to 60s to match that of the official Gravity Bridge.
```

Move this suggestion to under the unreleased section 👍